manifests/Waterfox/Waterfox/2020.05.yaml

```
@@ -0,0 +1,17 @@
+Id: Waterfox.Waterfox
+Name: Waterfox
+Version: 2020.05
+Publisher: Waterfox
+AppMoniker: waterfox
+License: Mozilla Public License
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0/
+Homepage: https://www.waterfox.net/
+Description: Browse the web your way, with the indie web browser.
+InstallerType: EXE
+Installers:
+  - Arch: x64
+    Url: https://cdn.waterfox.net/releases/win64/installer/Waterfox%20Current%202020.05%20Setup.exe
+    Sha256: f2549d077f0b6a9fb2c4c789e90a9a137583cbb02ad99be21faaa7c15ee90a11
+    Switches:
+      Silent: /S
+      SilentWithProgress: /S
```

```suggestion
Description: Browse the web your way, with the indie web browser.
InstallerType: nullsoft
Installers:
  - Arch: x64
    Url: https://cdn.waterfox.net/releases/win64/installer/Waterfox%20Current%202020.05%20Setup.exe
    Sha256: f2549d077f0b6a9fb2c4c789e90a9a137583cbb02ad99be21faaa7c15ee90a11
```

waterfox uses a nullsoft/nsis installer per https://www.reddit.com/r/waterfox/comments/309yj1/feature_request_msi_installer/

---

Thank you very much!